pkg/crc/config/viper_config.go

```diff
@@ -36,7 +39,33 @@ func NewViperStorage(configFile, envPrefix string) (*ViperStorage, error) {
 	}
 	// only watch config file if not running in a terminal/interactively
 	if !terminal.IsTerminal(int(os.Stdout.Fd())) {
-		v.WatchConfig()
+		go func() {
+			var tempConfig map[string]interface{} = make(map[string]interface{})
```

Comment:
why not just `var tempConfig = make(map[string]interface{})`

Reply:
Changed